airbyte-integrations/connectors/source-s3/metadata.yaml

```
@@ -10,7 +10,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 69589781-7828-43c5-9f63-8925b1c1ccc2
-  dockerImageTag: 4.1.4
+  dockerImageTag: 4.1.5
```

```suggestion
  dockerImageTag: 4.2.0
```
Shouldn't this be a minor bump as its a new "feature"?